src/collect/collector.rs

```diff
@@ -101,6 +105,22 @@ impl Group {
         Ok(self)
     }
 
+    /// Setup user defined input filter.
+    fn setup_filters(&mut self, collect: &Collect) -> Result<()> {
+        if let Some(f) = &collect.args()?.packet_filter {
+            let fb = FilterPacket::from_string(f.to_string())?;
+            self.probes.register_filter(
+                "packet".to_string(),
+                Filter::Packet(BpfFilter {
+                    prog: fb.to_bytes()?,
+                    target: "packet_filter".to_string(),
```

Comment:
so the idea is to do it inside replace_filters() based on the filter type.
I'm ok with that. Anyways this would make "target" here redundant and can be removed from BpfFilter, right?